settings/blt.settings.php

```diff
@@ -92,15 +92,11 @@
  * Site directory detection.
  */
 if (!isset($site_path)) {
-  try {
-    $site_path = DrupalKernel::findSitePath(Request::createFromGlobals());
-  }
-  catch (BadRequestHttpException $e) {
-    $site_path = 'sites/default';
-  }
+  $site_path = \Drupal::service('site.path');
```

Comment:
Why is this change necessary?

Reply:
I inadvertently put the notes about this in #3172 rather than this one since the two PRs are dependent on one another: Update site path detection to use core site.path service due to findSitePath bug always returning first default directory (see https://www.drupal.org/project/drupal/issues/2280409).